src/fragmentarium/ui/SearchForms.tsx

```diff
@@ -0,0 +1,230 @@
+    return (
+      <>
+        <Form>
+          <Form.Group as={Row} controlId="number">
+            <Col sm={{ span: 10, offset: 2 }}>
+              <Form.Control
+                type="text"
+                name="number"
+                value={this.state.number || ''}
+                placeholder="Search museum, accession, or CDLI number"
+                aria-label="Number"
+                onChange={(
+                  event: React.ChangeEvent<HTMLTextAreaElement>
+                ): void => this.onChange('number')(event.target.value)}
+              />
+            </Col>
+          </Form.Group>
+          <Form.Group as={Row} controlId="reference">
+            <Col
+              sm={2}
+              as={Form.Label}
+              className="TransliterationSearchForm__label"
+            >
+              <HelpTrigger overlay={this.ReferenceSearchHelp()} />
+            </Col>
+            <Col>
+              <BibliographySelect
+                aria-labelledby={'BibliographyTitle'}
+                value={this.state.referenceEntry}
+                onChange={this.onChangeBibliographyReference}
+                searchBibliography={(query) =>
+                  this.props.fragmentService.searchBibliography(query)
+                }
+              />
+            </Col>
+            <Col sm={5}>
+              <Form.Control
+                type="text"
+                name="pages"
+                placeholder="Page"
+                aria-label="Pages"
+                value={this.state.pages || ''}
+                onChange={(event: React.ChangeEvent<HTMLInputElement>): void =>
+                  this.onChangePages(event.target.value)
+                }
+                isInvalid={!this.state.isValid}
+              />
+              <Form.Control.Feedback type="invalid">
+                &quot;Page&quot; should be numeric.
+              </Form.Control.Feedback>
+            </Col>
+          </Form.Group>
+          <Form.Group as={Row} controlId="transliteration">
+            <Col
+              sm={2}
+              as={Form.Label}
+              className="TransliterationSearchForm__label"
+            >
+              <HelpTrigger overlay={this.TransliterationSearchHelp()} />
+            </Col>
+            <Col sm={10}>
+              <Form.Control
+                as="textarea"
+                value={this.state.transliteration || ''}
+                rows={Math.max(2, rows)}
+                placeholder="Search transliterations"
+                aria-label="Transliteration"
+                name="transliteration"
+                onChange={(
+                  event: React.ChangeEvent<HTMLTextAreaElement>
+                ): void => this.onChange('transliteration')(event.target.value)}
+              />
+            </Col>
+          </Form.Group>
+        </Form>
+        <ButtonToolbar>
+          <Col sm={{ offset: 2 }}>
+            <Button
+              className="w-25 m-1"
```

Comment:
What's the purpose of `w-25`? It makes the button look weird.

It is also a bit odd that this button has `m-1`, but the two others do not.

Reply:
`w-25` adds padding to the button so the button is wider than the content (= text within the button, e.g. "search").  This makes all the button look more similar in width regardless of their content. I can change it (the search button will look smaller then the other two though.)

Reply:
I decided to leave it like that for now instead of changing it. It looks better in my opinion we can change it anytime afterwards